tests/resources/MST_mirror_2f_measurements.schema.yml

```diff
@@ -1,28 +1,26 @@
 %YAML 1.2
 ---
 title: Schema for mirror panel 2F measurements
-name: mirror_2f_measurement
-description: |-
-  Mirror panel 2F measurements determining
-  panel radius and panel PSF.
 version: 0.1.0
-
 base_schema: simpipe-schema
 base_schema_url: https://github.com/gammasim/workflows/schema/jsonschema.yml
 base_schema_version: 0.1.0
-
+name: mirror_2f_measurement
+description: |-
+  Mirror panel 2F measurements determining
+  panel radius and panel PSF.
 data:
   - type: datatable
     table_columns:
-      - name: mirror_id
+      - name: mirror_panel_id
         description: |-
           Mirror panel ID.
         required: true
         units: dimensionless
         type: string
-      - name: mirror_diameter
+      - name: mirror_panel_radius
```

Comment:
I think this is a mistake. I think this should stay the mirror panel diameter and on top of it you can have also the radius of curvature.

Reply:
Changing this to radius of curvature.